website/www/site/content/en/community/join-beam.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Join Beam Community"
+aliases:
+  - /community/
+  - /use/issue-tracking/
+  - /use/mailing-lists/
+  - /get-started/support/
+---
+
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Join Beam Community!
+
+We’re so glad you want to join the Apache Beam community. Here’s a pathway you can follow to help you get started. This is not the only way to join, but it’s the recommended progression so you’re successful at reaching your objectives.
+
+## Join the Conversation
+
+{{< community/join_beam_columns number-of-data="0" >}}
+
+## Contribute to the Project
+
+{{< community/join_beam_columns number-of-data="1" >}}
+
+## Become a committer
+
+{{< community/join_beam_columns number-of-data="2" >}}
+
+## Become a PMC member
+
+{{< community/join_beam_columns number-of-data="3" >}}
+
+## Still have questions?
+
+Send us a note to user@beam.apache.org OR mention@ApacheBeam on Twitter OR ask us in the #beam slack channel. We look forward to seeing you in the community!
```

Comment:
It looks like this was the copy from the TDD, but I think we should make some minor adjustments (I updated the TDD as well), and add some links:
```suggestion

Send us a note at <a href="mailto:dev@beam.apache.org">user@beam.apache.org</a> OR mention <a target="_blank" href="https://twitter.com/apachebeam?lang=en">@ApacheBeam</a> on Twitter OR ask us in the #beam slack channel. We look forward to seeing you in the community!
```